baseclasses/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "1.2.4"
+__version__ = "1.2.5"
```

Comment:
I think now that we have broken backwards compatibility, we should bump to `1.3.0` at a minimum.

Reply:
Yes I agree